### PR TITLE
Add arp-cam & arp-core driver modules

### PR DESCRIPTION
**Pull request by TSAnsari**
Signed-off-by: tansari <tansari@luxoft.com>

### CODE REVIEW

File: arp-cam.c [hunk 1]
```diff
@@ -0,0 +1,538 @@
+/******************************************************************************
+ *
+ *   Copyright (C) 2017-2018 Pelagicore AB. All rights reserved.
```
Comment (thenor) [inline arp-cam.c]: We should take the opportunity to change any copyrights referring to Pelagicore. Should be "Luxoft Sweden AB".

File: arp-core.c [hunk 2]
```diff
@@ -0,0 +1,195 @@
+/******************************************************************************
+ *
+ *   Copyright (C) 2017-2018 Pelagicore AB. All rights reserved.
```
Comment (thenor) [inline arp-core.c]: Same as for arp-cam.

**Review by thenor — APPROVED** [review]

**Review by frznlogic — COMMENTED** [review]
There's very little comments in the code, barely anything at all... Would be nice with at least function descriptions etc?

**Comment by thenor** [comment]
I agree with @frznlogic that it would be nice with more in-code documentation.

**Comment by TSAnsari** [comment]
@frznlogic and @thenor I agree, but this code is provided by ARP team. The purpose of this commit was to place the code that ARP team is developing in a public repo so that they can directly make changes here rather than in there own local repos which are not visible to us. The only part I have modified is the Makefile so that it is possible to integrate this module in PELUX and can be built in Jenkins. I don't have enough knowledge about the code myself to add comments, but once this change is merged, the ARP team will have access to this for changes and then we can ask them to update with comments.

**Comment by thenor** [comment]
Ok for the integration.

**Review by frznlogic — APPROVED** [review]

**Pull request merged**